src/h5web/vis-packs/core/containers/RawVisContainer.tsx

```diff
@@ -1,15 +1,22 @@
 import RawVis from '../raw/RawVis';
 import type { VisContainerProps } from '../../models';
-import { useDatasetValue } from '../hooks';
 import { assertDataset, assertNonNullShape } from '../../../guards';
+import VisBoundary from '../VisBoundary';
+import ValueFetcher from '../ValueFetcher';
 
 function RawVisContainer(props: VisContainerProps) {
   const { entity } = props;
   assertDataset(entity);
   assertNonNullShape(entity);
 
-  const value = useDatasetValue(entity);
-  return <RawVis value={value} />;
+  return (
+    <VisBoundary resetKey={entity}>
+      <ValueFetcher
+        dataset={entity}
+        render={(value) => <RawVis value={value} />}
+      />
+    </VisBoundary>
```

Comment:
I could leave an extra error/suspense boundary in `Visualizer` and keep `RawVisContainer` and `ScalarVisContainer` unchanged, but I don't like that other containers end up having two boundaries (with the one in `Visualizer` ending up unused).

I also thought it best to make all containers consistent by making use of `VisBoundary` and `ValueFetcher` in every one of them.